packages/common/src/hardhat/tasks/acrossPool.ts

```diff
@@ -8,27 +8,30 @@ task("deploy-across-pool", "Deploys an L1 across pool and whitelists it within t
   .addParam("lptokensymbol", "Symbol of the LP tokens deployed for the pool", undefined, types.string)
   .addParam("l1tokenaddress", "Address of the token on L1", undefined, types.string)
   .addParam("lpfeeratepersecond", "Scales the amount of pending fees per second paid to LPs", undefined, types.string)
+  .addParam("iswethpool", "Set if this is the across weth pool", undefined, types.string)
   .setAction(async function (taskArguments, hre_) {
     const hre = hre_ as CombinedHRE;
     const { deployments, getNamedAccounts, web3 } = hre;
     const { deploy } = deployments;
     const { deployer } = await getNamedAccounts();
-    const { lptokenname, lptokensymbol, l1tokenaddress, lpfeeratepersecond } = taskArguments;
+    const { lptokenname, lptokensymbol, l1tokenaddress, lpfeeratepersecond, iswethpool } = taskArguments;
 
     const BridgeAdmin = await deployments.get("BridgeAdmin");
     const bridgeAdmin = new web3.eth.Contract(BridgeAdmin.abi, BridgeAdmin.address);
     console.log(`Loaded BridgeAdmin @ ${bridgeAdmin.options.address}`);
-    console.log(`Deploying bridgePool`);
```

Comment:
Why not make this script a `deploy` script?

Reply:
need params!